backend/database/models/_assoc_tables.py

```diff
@@ -0,0 +1,40 @@
+from .. import db
+from backend.database.models.officer import Rank
+
+incident_agency = db.Table(
+    'incident_agency',
+    db.Column('incident_id',  db.Integer, db.ForeignKey('incident.id'),
+              primary_key=True),
+    db.Column('agency_id', db.Integer, db.ForeignKey('agency.id'),
+              primary_key=True),
+    db.Column('officers_present', db.Integer)
+)
+
+incident_tag = db.Table(
+    'incident_tag',
+    db.Column('incident_id', db.Integer, db.ForeignKey('incident.id'),
+              primary_key=True),
+    db.Column('tag_id', db.Integer, db.ForeignKey('tag.id'), primary_key=True)
+)
+
+agency_officer = db.Table(
+    'agency_officer',
+    db.Column('agency_id', db.Integer, db.ForeignKey('agency.id'),
+              primary_key=True),
+    db.Column('officer_id', db.Integer, db.ForeignKey('officer.id'),
+              primary_key=True),
+    db.Column('earliest_employment', db.Text),
+    db.Column('latest_employment', db.Text),
+    db.Column('badge_number', db.Text),
+    db.Column('unit', db.Text),
```

Comment:
Just thinking about this - should units be a separate table which relates to an agency? This would help ensure that the unit names would be consistent across relationships which would make identifying problematic units more consistent.